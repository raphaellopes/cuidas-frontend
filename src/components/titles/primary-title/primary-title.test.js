// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import PrimaryTitle from './index';

const component = props => (renderer.create(
  <PrimaryTitle {...props} />,
));

describe('<PrimaryTitle />', () => {
  test('Should render the component', () => {
    const tree = component({ children: 'my children' }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
