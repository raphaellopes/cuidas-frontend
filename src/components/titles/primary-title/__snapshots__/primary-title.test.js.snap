// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrimaryTitle /> Should render the component 1`] = `
<h1
  className="sc-bdVaJa noaWR"
>
  my children
</h1>
`;
